website/content/docs/internals/telemetry/metrics/raft.mdx

```diff
@@ -0,0 +1,175 @@
+---
+layout: docs
+page_title: "Telemetry reference: Raft metrics"
+description: >-
+  Technical reference for integrated storage telemetry values.
+--- 
+
+# Raft telemetry
+
+Raft telemetry provides information on the raft consensus algorithm used for
```

Comment:
```suggestion
Raft telemetry provides information on 
```
It's a little weird to me the way this was phrased, proposing a simpler alternative.